src/locales/fr/config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "native": "English",
```

Comment:
This should probably be Française (and French below).

Reply:
Fixed